service/service.go

```diff
@@ -0,0 +1,24 @@
+package service
+
+import (
+	"Price_Notification_System/producer/store"
+)
+
+// HistoricalStore - type related to the store.Storage interface
+type HistoricalStore struct {
```

Comment:
the types `HistoricalStore` and `AlertStore` would be a part of the store package, and would implement the store interface..